lib/util/compat.h

```diff
@@ -0,0 +1,12 @@
+#ifndef __COMPAT_H
+#define __COMPAT_H
+
+#ifndef HAVE_LIBBPF_BTF__TYPE_CNT
+static __u32 btf__type_cnt(const struct btf *btf)
```

Comment:
I was wondering if there is a specific reason why you decided to inline them for libxdp, and not add them here?

Reply:
Because the code in `lib/libxdp` is supposed to be self-contained, so we can't include anything from `util/` in there...